pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -80,6 +75,12 @@ buildPythonApplication rec {
   outputs = [ "out" "terminfo" "shell_integration" ];
 
   patches = [
+    # Fix clone-in-kitty not working on bash >= 5.2
```

Comment:
```suggestion
    # Fix clone-in-kitty not working on bash >= 5.2. Can be removed on kitty release > 0.26.5.
```

Reply:
I don't think we need to add this since the patch will fail to apply when updating to a new release that includes that commit (alerting whoever does the update that the patch needs to be removed).

Reply:
A failure doesn't necessarily mean that the patch is included in the next release. It's much simpler to read this message than troubleshoot a failing patch.

Reply:
Done.